### DISCUSSION TITLE
Nested include statements only return rows with relational data available

### DISCUSSION THREAD

**Discussion by NatsumeKaito** [prisma/prisma#10648] [Q&A]
Hello guys, I need to return a relation tables data with nested include statement, but the query I have only returned rows where the relational table has data.

for example, i have 3 rows, two of them have data related to the other tables and one doesn't

ID   COLUMN1 COLUMN 2 COLUMN3 RELATEDCOLUMN1
1       DATA         DATA             DATA         IDRELATEDATA  <-- is  included in the results
2      DATA         DATA             DATA         IDRELATEDATA  <-- is  included in the results
3       DATA         DATA             DATA         NULL   <-- is not included in the results

this is my query:

```ts
await prisma.virtual_Room.findMany({
  take: limit,
  skip: startIndex,
  orderBy: {
      created_at: 'desc',
  }, 
  include: {
      room: {
          include: {
              games: true,
              levels: true
          }
      },
      room_user: {
          include: {
              users: {
                  select: {
                      username: true
                  }
              }
          }
      },
  },
  where: {
      room: {
          type
      }
  }
})
```

thanks in advance

**Comment by austincrim** [comment]
Hey @NatsumeKaito 👋 !

Would you mind providing your `schema.prisma` and some sample data so I can try to recreate this?
